terraform/state-management/variables.tf

```diff
@@ -0,0 +1,23 @@
+# tf variables
+# project to create the related resources in
+variable "project" {
+  type    = string
+  default = "cuhealthai-sandbox"
+}
+# Region to be used with the project resources
+variable "region" {
+  type    = string
+  default = "us-central1"
```

Comment:
I thought that we were using a European instance?

Reply:
Sorry if I'm butting in, but this bucket is for managing terraform's shared state, i.e. keeping track of what cloud resources it created so it can generate diffs when the terraform configuration changes; the data here is small, kilobytes in size, and will probably only be managed on our (i.e. DBMI's) end. It's not the actual data that the project is providing, which AFAIK is in a bucket in Europe.

Reply:
Thanks @gwaybio and @falquaddoomi - what Faisal mentions is correct. We can decouple the bucket location where Assay.Works uploads data from the state management bucket. I changed the location to be thorough with potential cost savings with egress from an EU location to North America (otherwise the general storage costs are I believe the same with `europe-west4` and `us-central1`). The cost differences are very minimal, so this is mostly about a practice/mindset of cost avoidance in relation to geographic location. I'll leave the location as-is with this in mind unless there are major concerns here.